src/SelectField/SelectFieldLabel.jsx

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import DropDownArrow from '../svg-icons/navigation/arrow-drop-down';
+
+const getStyles = (props) => {
+  const {
+    disabled,
+    style,
+  } = props;
+
+  const {palette, dropDownMenu} = props.muiTheme;
+  const {accentColor} = dropDownMenu;
+  const {marginTop} = style || {};
+
+  return {
+    label: {
+      color: disabled ? palette.disabledColor : palette.textColor,
+      cursor: disabled ? 'not-allowed' : 'pointer',
+      opacity: 1,
+      position: 'absolute',
+      paddingTop: (marginTop || 0) + 12, // just like magic
+      width: '100%',
+      height: '100%',
+      outline: 'none',
+    },
+    icon: {
+      fill: accentColor,
+      position: 'absolute',
+      right: 0,
+      top: 14,
+    },
+  };
+};
+
+                        // this can't be stateless
+                        // as we want to call focus on it
+                        // react-warpgate may help
+const SelectFieldLabel = React.createClass({
```

Comment:
how can you focus a label? react-warpgate is for warping methods calls **down** not **through**. I don't think react-warpgate is useful here. maybe we should follow the pattern used in Popover and take a focus target instance :grin: